spec/app/models/metasploit_data_models/automatic_exploitation/match_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe MetasploitDataModels::AutomaticExploitation::Match do
+  it_should_behave_like 'Metasploit::Concern.run'
+
+
+  describe "associations" do
```

Comment:
Missing general association expectation using `it { is_expected.to belong_to ... }`.